Automattic/vip-go-mu-plugins
# Moves versioning hook from plugin_loaded to init

## Description

ElasticPress loads features which includes registering of some indexables on `init` hook rather than `

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Create post - ` wp post create --post_title="test" --post_content="test" --post_excerpt="test" --post_status="publish"`
1. Add versiont 2 `wp vip-search  index-versions add post`
1. Index the second version `wp vip-search index --version=2 --indexables=post --setup`
1. Check that the post is in ES - `wp vip-search documents get post <id>` 
1. activate 2nd version - `wp vip-search index-versions activate post 2`
1. Delete the post  - ` wp post delete <id>`
1. Check that the post is NOT in v2 index - `wp vip-search documents get post <id>` 
1. Switch back to version 1 - `wp vip-search index-versions activate post 1`
1. Check that the post is NOT in v1 index as well - `wp vip-search documents get post <id>`
